types/pm-rpc.d.ts

```diff
@@ -0,0 +1,42 @@
+export interface TargetDefinition {
+  target?: MessagePort | Worker | Window | HTMLIFrameElement;
+}
+
+export declare type OnApiCall = (data: {
+  appId: string;
+  call: string;
+  args: any[];
+}) => void;
+
+/**
+ * Set global api that will be consumed
+ * @param {string} apiId Id of the api that will be requested
+ * @param {object} api API that get exposed
+ * @param {function} onApiCall Deprecated - callback when some api function is called
+ * @param workers - array of Workers that can consume API
+ */
+export declare const set: <T extends unknown>(
+  apiId: string,
+  api: T,
+  {
+    onApiCall,
+    workers,
+  }?: {
```

Comment:
`}?: {` feels clumsy. How about this?

```suggestion
  options?: {
```

Reply:
It was generated from sources intially